README.md

```diff
@@ -1060,26 +1060,78 @@ view heimdall                 Run an instance of Heimdall Lite to
 
 #### Summary
 
-To get a quick compliance summary from an HDF file (grouped by profile name) use the `saf view summary` command:
+Generate a comprehensive summary of compliance data, including totals and counts, from your HDF files.
 
-```
-view summary                  Get a quick compliance overview of an HDF file
+The output can be displayed in the console, or exported as YAML, JSON, or a GitHub-flavored Markdown table.
 
-  USAGE
-    $ saf view summary -i <hdf-file> [-h] [-j] [-o <output>]
+```powershell
```

Comment:
I didn't catch it - what is the difference in display for the `shell` or `powershell` code blocks? That way I know when to use which

Reply:
Just did this update in the Summary example for now. 

To not 'bloom' the PR too much, I will add that change to the rest after we merge this change.